vsts/test-release-nuget.yaml

```diff
@@ -172,6 +172,8 @@ jobs:
       # Change maxParallel to 1 make builds run in serial rather than in parallel
       maxParallel: 100
       matrix:
+        .Net 6.0:
+          FRAMEWORK: net6.0
```

Comment:
Do we need to make this same change in our nightly build yaml as well? Or are we already testing .NET 6.0 there?

Reply:
Will make that update in the next change.